setup.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
You can list all of this information in `pyproject.toml`, if it's not already there. This file shouldn't be necessary.

Reply:
Yes; this is one of the problems (needing a complicated `setup.py` file) that Poetry is explicitly intended to solve. Although this file isn't too complicated, for an actual package to be published on PyPI, there might be significantly more configuration needed. For guidance on what and where to put metadata, see Poetry's [documentation](https://python-poetry.org/docs/pyproject/), which lists every valid Poetry parameter you can add to `pyproject.toml`.